assets/scss/_gallery.scss

```diff
@@ -52,17 +55,15 @@
   }
 
   &__photo {
-    display: flex;
-    margin-left: 20rem;
+    width: 100%;
 
-    @media (max-width: $mid-size) {
-      align-items: center;
-      margin-left: 0;
-    }
   }
 
   &__photo-container {
-
+    display: flex !important;
```

Comment:
zawsze warto dopisać komentarz - ładniej to będzie wyglądało 

// to overwrite slick-slider styles